test/cluster/test_read_repair.py

```diff
@@ -259,64 +193,51 @@ async def test_incremental_read_repair(data_class: DataClass, workdir: str, mana
     logger.info(f"random-seed: {seed}")
     random.seed(seed)
     cmdline = ["--hinted-handoff-enabled", "0",
+               "--cache-hit-rate-read-balancing", "0",
                "--query-tombstone-page-limit", "10",
                "--query-page-size-in-bytes", "1024"]
     node1 = await manager.server_add(cmdline=cmdline)
     node2 = await manager.server_add(cmdline=cmdline)
```

Comment:
better use `manager.servers_add(2, , auto_rack_dc="dc1")` to build the cluster

Reply:
Done in v3.